mautrix/types/event/message.py

```diff
@@ -272,12 +280,13 @@ class LocationInfo(SerializableAttrs):
 
 
 @dataclass
-class MediaMessageEventContent(BaseMessageEventContent, SerializableAttrs):
+class MediaMessageEventContent(FormatableMessageEventContent, SerializableAttrs):
```

Comment:
This should just inherit TextMessageEventContent instead of adding a new class, the same helper methods are necessary anyway.